README.md

```diff
@@ -1,229 +1,67 @@
 
-The following will download all Blob container contents into the local file system. `mycontainer` directory will not be created in the destination because the globbing pattern looks for all paths inside `mycontainer` in the source rather than the `mycontainer` container itself.
-```
-./azcopy cp "https://myaccount.blob.core.windows.net/mycontainer/*?sastokenhere" /data/ --recursive=true
-```
+:white_check_mark: Recover from failures by restarting previous jobs.
 
-The following command will download all txt files in the source to the `directory1` path. Note that AzCopy will scan the entire source and filter for `.txt` files. This may take a while when you have thousands/millions of files in the source.
-```
-./azcopy cp "https://myaccount.blob.core.windows.net/mycontainer/directory1/*.txt?sastokenhere" /data/directory1
-```
+## Find help
 
-### Copy data between Azure Storage accounts (currently supports Block Blobs only)
+For complete guidance, visit any of these articles on the docs.microsoft.com website.
 
-Copying data between two Azure Storage accounts make use of the PutBlockFromURL API, and does not use the client machine's network bandwidth. Data is copied between two Azure Storage servers. AzCopy simply orchestrates the copy operation.
-```
-./azcopy cp "https://myaccount.blob.core.windows.net/?sastokenhere" "https://myotheraccount.blob.core.windows.net/?sastokenhere" --recursive=true
-```
+:eight_spoked_asterisk: [Get started with AzCopy](https://docs.microsoft.com/azure/storage/common/storage-use-azcopy-v10)
 
-### Advanced Use Cases
+:eight_spoked_asterisk: [Transfer data with AzCopy and blob storage](https://docs.microsoft.com/azure/storage/common/storage-use-azcopy-blobs)
 
-#### Configure Concurrency
+:eight_spoked_asterisk: [Transfer data with AzCopy and file storage](https://docs.microsoft.com/azure/storage/common/storage-use-azcopy-files)
 
-Set the environment variable `AZCOPY_CONCURRENCY_VALUE` to configure the number of concurrent requests. This is set to 300 by default. Note that this does not equal to 300 parallel connections. Reducing this will limit the bandwidth, and CPU used by AzCopy.
+:eight_spoked_asterisk: [Transfer data with AzCopy and Amazon S3 buckets](https://docs.microsoft.com/azure/storage/common/storage-use-azcopy-s3)
 
-#### Configure proxy settings
-To configure the proxy settings for AzCopy v10, set the environment variable https_proxy using the following command:
+:eight_spoked_asterisk: [Configure, optimize, and troubleshoot AzCopy](https://docs.microsoft.com/azure/storage/common/storage-use-azcopy-configure)
 
-```
-# For Windows:
-set https_proxy=<proxy IP>:<proxy port>
-# For Linux:
-export https_proxy=<proxy IP>:<proxy port>
-# For MacOS
-export https_proxy=<proxy IP>:<proxy port>
-```
+### Find help from your command prompt
 
-For proxy authentication, use the following format:
+Be sure to add AzCopy to your system path. Then, open a command prompt window.
```

Comment:
I'm in two minds about this. Technically, adding it to the path is not strictly necessary. But, in PowerShell at least, if its not in the path you need to invoke it as .\azcopy instead of just azcopy.

As a user, I never like being told I _have_ to add something to my path. But I can see how it simplifies the rest of the doc if they do add it. What do you think?

Reply:
Thanks John. Perhaps we can tell them to "optionally" add this to their path. With a path variable in place, the use of "azcopy" at the command prompt becomes universal. ".\azcopy" is Windows PowerShell specific. This won't work for a standard command prompt and I'm not sure about Linux environments. Perhaps we can add a note to the core docs content that if they don't add this to their path, and they are in a PowerShell environment, they will have to preface with ".\"?

Reply:
Ok - I added some text for the optional system path and for PowerShell prompts.